### ISSUE TITLE
Add episode python script does not work

### ISSUE THREAD

**Issue by PhyberApex** [PhyberApex/homeassistant-config#36]
```
2017-07-06 05:17:48 ERROR (Thread-5) [homeassistant.components.python_script.add_new_episode.py] Error executing script: 'NoneType' object is not callable
Traceback (most recent call last):
  File "/opt/homeassistant/venv/lib/python3.4/site-packages/homeassistant/components/python_script.py", line 107, in execute
    exec(compiled.code, restricted_globals, local)
  File "add_new_episode.py", line 7, in <module>
TypeError: 'NoneType' object is not callable
```

**Comment by PhyberApex** [comment]
Check initial writing of sensor

**Comment by PhyberApex** [comment]
Fixed with #38
